packages/server-wallet/src/wallet/index.ts

```diff
@@ -74,6 +74,12 @@ export class Wallet implements WalletInterface {
     return participant;
   }
 
+  public async getSigningAddress(): Promise<string> {
```

Comment:
I noticed that the wallet already has a `getParticipant` method. I think that method should just create a signing wallet if none exists. That removes the need to expose `getSigningAddress`.

Even better: the wallet needs to know what my participant Id is. So, the application should call something like `setParticipant({participantId: string, destination: string})` and it should:
- error if that participant is already set (uniqueness constraint on `participantId`
- create a random signing key for that participant, save that in a `SigningWallet`, and also save the participantId/destination somewhere. (Maybe the `SigningWallet` model is actually a `Participant` model?)